include/tlsuv/tls_engine.h

```diff
@@ -200,6 +200,17 @@ struct tls_context_s {
      */
     int (*set_own_cert)(tls_context *ctx, tlsuv_private_key_t key, tlsuv_certificate_t cert);
 
+    /**
+     * Allows partial chain matching.
+     *
+     * Causes intermediate certificates in the trust store to be treated as trust-anchors,
+     * in the same way as the self-signed root CA certificates.
+     * @param ctx
+     * @param on
```

Comment:
assuming this should return an error code for mbedtls. how will that work? should there be an mbedtls implementation or does it somehow happen without one?